modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -313,6 +312,7 @@ protected void containerIsStarted(InspectContainerResponse containerInfo) {
      *
      * @return a new Remote Web Driver instance
      */
+    @Deprecated
```

Comment:
```suggestion
     *
     * @return a new Remote Web Driver instance
     * @deprecated use {@link #getSeleniumAddress()} instead
     */
    @Deprecated
```

I eyeballed the Javadoc for the `@deprecated` annotation.